app/views/pages/home/_location_categories.html.haml

```diff
@@ -0,0 +1 @@
+%h1.govuk-heading-xl Location Categories
```

Comment:
Note, we haven't used any `I18n` yet, but this should be fixed in future tickets.

Reply:
That's fine 👍